File: Server/PRMServer.Application/DTOs/Users/UserSummaryDTO.cs
namespace PRMServer.Application.DTOs.Users
{
    public record UserSummaryDTO
    {
        public required int Id { get; set; }
        public required string Username { get; set; }
        public required string Email { get; set; }
        public required int DecksCount { get; set; }
    }
}

File: App/app/src/main/java/com/prm392g2/prmapp/dtos/users/UserSummaryDTO.java
package com.prm392g2.prmapp.dtos.users;

public class UserSummaryDTO {
    public int Id;
    public String Username;
    public int DecksCount;

    public int getId() {
        return Id;
    }

    public void setId(int id) {
        Id = id;
    }

    public String getUsername() {
        return Username;
    }

    public void setUsername(String username) {
        Username = username;
    }

    public int getDecksCount() {
        return DecksCount;
    }

    public void setDecksCount(int decksCount) {
        DecksCount = decksCount;
    }
}
